src/app/modules/core/components/jobsons/jobsons.component.ts

```diff
@@ -167,33 +170,59 @@ export class JobsonsModalComponent implements OnInit {
     } else {
       this.dateModel = new Date(this.dateModel);
     }
-    this.TravelTimeService.ExecuteJobson(this.StudyService.selectedStudy.SpillMass, this.dateModel.toISOString(), this.reachList)
+    let postReachList = [];
+    this.reachList.forEach(reach => {
+      reach.parameters.splice(6,1);
+      postReachList.push(reach);
+    })
+    this.TravelTimeService.ExecuteJobson(this.StudyService.selectedStudy.SpillMass, this.dateModel.toISOString(), postReachList)
       .toPromise().then(data => {
-        console.log(data);
         this.StudyService.selectedStudy.Results = data;
         this.StudyService.SetWorkFlow("totResults", true);
         this.gettingResults = false;
+        this.activeModal.dismiss();
       })
       .catch((err) => {
         console.log("error: ", err.message);
         this.sm(err, "Time of Travel Services")
         this.gettingResults = false;
       });
   }
+
+  public openCloseItem(index): void {
+    this.currentIndex = index;    
+    if(this.reachList[this.currentIndex].parameters.show) {
+      this.reachList[this.currentIndex].parameters.show = false;
+    } else {
+      if(this.lastIndex !== null) {
+        this.reachList[this.lastIndex].parameters.show = false;
+      }
+      this.reachList[this.currentIndex].parameters.show = true;
```

Comment:
If your lastIndex is the same as current index this part will close and open the same panel on click. You have to check if current, last and a new one are the same.

Reply:
Most probable that this was a reason for error if there was any

Reply:
currentIndex has been changed to selectedIndex